From fc481085c58c1add1bc1b6980c4e0713964e06a9 Mon Sep 17 00:00:00 2001
From: Frajo Haider <f_haider@gmx.at>
Date: Wed, 22 Jan 2020 11:13:13 +0200
Subject: [PATCH 06/18] Revert "linux-user: Use safe_syscall wrapper for
 connect syscall"

This reverts commit 2a3c7619288af9cfcc09a233dce911bf80849dfb.
---
 linux-user/syscall.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/linux-user/syscall.c b/linux-user/syscall.c
index 9a95158..d5bc5b8 100644
--- a/linux-user/syscall.c
+++ b/linux-user/syscall.c
@@ -2907,7 +2907,7 @@ static abi_long do_connect(int sockfd, abi_ulong target_addr,
     if (ret)
         return ret;
 
-    return get_errno(safe_connect(sockfd, addr, addrlen));
+    return get_errno(connect(sockfd, addr, addrlen));
 }
 
 /* do_sendrecvmsg_locked() Must return target values and target errnos. */
-- 
1.8.3-rc3

